docker-compose.yml

```diff
@@ -0,0 +1,121 @@
+version: "1.0"
+
+services:
+  zookeeper:
+    image: bitnami/zookeeper:3.8.0
+    ports:
+      - 2181:2181
+    environment:
+      - ZOO_PORT_NUMBER=2181
+      - ZOO_TICK_TIME=2000
+      - ALLOW_ANONYMOUS_LOGIN=yes
+    networks:
+      - devenv
+    
+  kafka:
+    image: bitnami/kafka:3.1.0
+    ports:
+      - "9092:9092"
+    environment:
+      - KAFKA_BROKER_ID=0
+      - ALLOW_PLAINTEXT_LISTENER=yes
+      - "KAFKA_HEAP_OPTS=-Xmx500m -Xms500m"
+      - KAFKA_CFG_ZOOKEEPER_CONNECT=zookeeper:32181
```

Comment:
Changed this to port 2181 as specified on line 9 and added dependency to zookeper to get it work on intel machine
```
    depends_on:
      - zookeeper
```